src/pages/AddLiquidity/index.tsx

```diff
@@ -511,9 +525,9 @@ export default function AddLiquidity({
             </RowBetween>
           )}
         {mustCreateSeparately && (
-          <ButtonError onClick={onCreate}>
+          <ButtonError onClick={() => setShowConfirm(true)} disabled={!price}>
             <Text fontWeight={500}>
-              <Trans>Create</Trans>
+              <Trans>{!price ? 'Enter Starting Price' : 'Create'}</Trans>
```

Comment:
Probably need to split up the strings to make extraction/translation easier.

```suggestion
              {!price ? <Trans>Enter Starting Price</Trans> : <Trans>Create</Trans>}
```